### PR TITLE
update to cerc-io/laconicd

### PR DESCRIPTION
**Pull request by 0xmuralik**
Closes: #32 

## Description

Update package names, dependencies and docker/ReadMe files to match with cerc-io/laconicd

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

**Review by i-norden — COMMENTED** [review]
LGTM, thanks for prioritizing this last minute. Looks like linting is still failing but that is being addressed in another PR. Gosec is also failing and I'm not sure if that is some weird non-deterministic finickiness or something else (error seems to be "[gosec] 2022/09/07 11:48:49 strconv.ParseUint: parsing "64-69": invalid syntax"), if you can look into that before merging this that would be great.

And there has been some back-and-forth discussion about what to name this repo but let's stick with laconicd for now and if we need to change it later on we can.

File: rpc/ethereum/backend/backend.go [hunk 1]
```diff
@@ -131,7 +131,7 @@ func (e *EVMBackend) BlockNumber() (hexutil.Uint64, error) {
 		return 0, fmt.Errorf("unexpected '%s' gRPC header length; got %d, expected: %d", grpctypes.GRPCBlockHeightHeader, headerLen, 1)
 	}
 
-	height, err := strconv.ParseUint(blockHeightHeader[0], 10, 64)
+	height, err := strconv.ParseUint(blockHeightHeader[0], 10, 63)
```
Comment (0xmuralik) [inline rpc/ethereum/backend/backend.go:134]: See https://github.com/cosmos/gosec/tree/master/rules/sdk#strconv-unsigned-integers-cast-to-signed-integers-overflow

**Comment by 0xmuralik** [comment]
Still Failing after adding the change: https://github.com/cerc-io/laconicd/pull/35/files/9c49c2b934b08ceff409d3dcc08ee13b5606148e..9e014725b8bf486c3ab62e67191c370fc03a1952#diff-7f75fec163267a96892043b684f88d495897981a15943b06e4b11e5458c62b8c

Gosec rule for parsing intergers: https://github.com/cosmos/gosec/tree/master/rules/sdk#strconv-unsigned-integers-cast-to-signed-integers-overflow

File: rpc/ethereum/backend/backend.go (hunk 1 above)
Comment (i-norden) [reply rpc/ethereum/backend/backend.go:134]: That's odd, thanks for checking. Let's just take note of this for now and move on.

**Review by i-norden — APPROVED** [review]

**Pull request merged**